apps/laboratory/src/components/AppKitButtonsMultiChain.tsx

```diff
@@ -109,7 +109,11 @@ export function AppKitButtonsMultiChain() {
                   <Button
                     isDisabled={!isAnyAccountConnected}
                     data-testid="disconnect-hook-button"
-                    onClick={() => disconnect()}
+                    onClick={() => {
+                      disconnect({ namespace: 'eip155' })
```

Comment:
Though we're disconnecting all namespaces if we don't pass any parameters to it? Isn't working like that?